.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "markdown_metayaml"]
```

Comment:
What I saw was [this line](https://github.com/getpelican/pelican/blob/master/pelican/readers.py#L219) checking if `"authors"` was in `METADATA_PROCESSORS` (it is, along with tags), then checking the length and (for more than 1 author or tag) complaining about multiple definitions and only using the first author/tag in the list.